engine/order_manager.go

```diff
@@ -875,3 +959,46 @@ func (s *store) getFilteredOrders(f *order.Filter) ([]order.Detail, error) {
 	}
 	return os, nil
 }
+
+// getActiveOrders returns copy of the orders that are active
```

Comment:
If you have time could you please add a test for this and for the functions you have added in ```order_manager.go in``` ```order_manager_test.go```. Let me know if you can, if you can't, I can add a public trello Item.

Reply:
OK I added them, let's see if it's sufficient :)
I made 2 code changes based on the test cases, good!